api/envoy/extensions/common/dynamic_forward_proxy/v3/dns_cache.proto

```diff
@@ -53,7 +53,7 @@ message DnsCacheConfig {
   // likely build a "happy eyeballs" connection pool which would race the primary / fall back
   // address and return the one that wins. This same method could potentially also be used for
   // QUIC to TCP fall back.]
-  config.cluster.v3.Cluster.DnsLookupFamily dns_lookup_family = 2
+  clusters.dns.v3.DnsConfig.DnsLookupFamily dns_lookup_family = 2
```

Comment:
There should be no changes in this file.  We cannot change the types of existing fields, since that's a breaking change.